src/components/TakeAction.vue

```diff
@@ -165,8 +174,10 @@ export default {
       return this.$store.state.selectedRep
     }
   },
+  mounted() {},
   methods: {
     nextPage(attrs) {
+      this.$refs.letterLoad.persistCustomization()
```

Comment:
@DietBepis1: While testing the custom message flow, I see an error 
```this.$refs.letterLoad.persistCustomization is not a function`` and I am reading up on how to fix this. Please let me know if you have any suggestions. Thank you.

Reply:
I think we can just get rid of this now that we are using the watcher.

Reply:
Removed it.